samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,22 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Energy consumption and the number of measurements performed in a given time depend on:
+
+   * Advertiser settings.
+   * Scanner settings.
+   * Extended size of measurement window by ``extra_window_time_us`` in the :c:struct:`dm_request` structure.
+
+The shorter the interval, the higher the frequency of broadcasting advertising packages.
+This leads to a higher probability of synchronization and to a higher number of measurements taken at a given time.
+However, the higher number of packets transmitted also results in higher power consumption.
```

Comment:
maybe we can add a note here saying that this decreases the latency until a new device is picked up and that.
Additionally, maybe we can link the relevant KConfigs here.

Reply:
Added additional note.